watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_sweep.py

```diff
@@ -91,11 +91,16 @@ def run_analysis(case_num, nx, interpolate_nan_outputs=True):
         raise ValueError("case_num = %d not recognized." % (case_num))
 
     output_filename = "sensitivity_" + str(case_num)
+    output_path = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        output_filename,
+    )
+
     global_results = parameter_sweep(
         m,
         sweep_params,
         outputs,
-        csv_results_file_name=output_filename,
+        csv_results_file_name=output_path,
```

Comment:
Can this sweep file be applied to the flowsheet w/WWTP and RO? Also, are the high-level metric the same between the 2 flowsheets if we wanted run comparisons between them?

Reply:
I don't think the sweep file can do both, as of now. My plan was to make another sweep file for RO but I can integrate them instead. This specific addition was slipped in from a previous comment and ensures that the sweep files are saved in the same directory as the flowsheet. I'll need to adjust the costing for dye in the previous analysis slightly, but we should be able to compare the levelized cost on a treated feed water basis.

Reply:
OK- we can possibly get around to having one sweep file that could be applicable to both flowsheets (e.g., by making sure cost metrics are symmetrical between flowsheets) in a subsequent PR.